data/helpers.d/network

```diff
@@ -0,0 +1,22 @@
+# Normalize the url path syntax
+# Handle the slash at the beginning of path and its absence at ending
+# Return a normalized url path
+#
+# example: url_path=$(ynh_normalize_url_path $url_path)
+#          ynh_normalize_url_path example -> /example
+#          ynh_normalize_url_path /example -> /example
+#          ynh_normalize_url_path /example/ -> /example
+#
+# usage: ynh_normalize_url_path path_to_normalize
+# | arg: url_path_to_normalize - URL path to normalize before using it
+ynh_normalize_url_path () {
+	path=$1
```

Comment:
In fact, it's not a problem, because this helper should be used in an sub shell. But, like we said before, it's better to avoid to use path. Here, I simply forgot to change it.